LINK/usr/bin/evmserver.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
-PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local:/usr/local/sbin:/usr/local/bin
```

Comment:
I believe this might have been added since the path did not have the typical locations when init used to launch this process.  If the path has these values when launched from systemd or in the container enviroment, I'm 👍